### PR TITLE
fix(google-maps-provider): quick fix google maps check

### PR DESCRIPTION
**Pull request by BiniCodes**
So this was sometimes occuring:

image

I think you wanted to handle this via `isLoadingAPI`, no? @plumdumpling 
This fixes it for now, but we got to recheck the loading of the script tag as the state of `isLoadingAPI` is false sometimes when loading the script and I think this is the main problem.

### CODE REVIEW

**Comment by BiniCodes** [comment]
Tests should work after merging: https://github.com/ubilabs/google-maps-react-hooks/pull/120

**Review by plumdumpling — CHANGES_REQUESTED** [review]
> I think you wanted to handle this via isLoadingAPI, no? @plumdumpling

Yes, the `isLoadingAPI` state should do this. Please let's see what's wrong with it and fix it there.

**Comment by BiniCodes** [comment]
> > I think you wanted to handle this via isLoadingAPI, no? @plumdumpling
> 
> Yes, the `isLoadingAPI` state should do this. Please let's see what's wrong with it and fix it there.

Added a comment above the code for the check as I currently do not see another way :)

File: library/src/google-maps-provider.tsx [hunk 1]
```diff
@@ -150,19 +151,24 @@ export const GoogleMapsProvider: React.FunctionComponent<
 
   // Handle Google Maps map instance
   useEffect(() => {
-    let newMap: google.maps.Map | undefined;
+    // Check for google.maps is needed because of Hot Module loading
```
Comment (plumdumpling) [inline library/src/google-maps-provider.tsx:154]:
```suggestion
    // Check for google.maps is needed because of Hot Module Replacement
```

**Review by plumdumpling — APPROVED** [review]

File: library/src/google-maps-provider.tsx (hunk 1 above)
Comment (BiniCodes) [reply library/src/google-maps-provider.tsx:154]: I was thinking about it :D

**Pull request merged**